extension/chrome/elements/compose.ts

```diff
@@ -83,7 +83,7 @@ Catch.try(async () => {
     draftId = storage.drafts_reply[threadId]; // there may be a draft we want to load
   }
 
-  const processedUrlParams = { acctEmail, draftId, threadId, subject, from, to, frameId, tabId, isReplyBox, skipClickPrompt, parentTabId, disableDraftSaving, debug };
+  const processedUrlParams = { acctEmail, draftId, threadId, threadMsgId, subject, from, to, frameId, tabId, isReplyBox, skipClickPrompt, parentTabId, disableDraftSaving, debug };
```

Comment:
The threadMsgId is used as a backup mechanism in case we could not parse threadId. In fact I want to remove & deprecate its use, but didn't have a chance to do it yet.

The issue is that it's not guaranteed to be there, you should go by threadId.

Reply:
Okay, I will remove `threadMsgId`. Thanks!

Reply:
in the code when we forward a message I can see you get last msgId using `await this.app.emailProviderDetermineReplyMsgHeaderVariables();`.
I think I can use this one too instead of getting all the messages in the thread. Am I correct?

Reply:
Yes, you can do it similarly to what forward does.